### PR TITLE
Box too big

### LINKED ISSUE #3824: Analytics warning is too wide for default Mac OS X terminal window size
screen shot 2016-05-09 at 16 25 55

### PR DESCRIPTION
**Pull request by sethladd**
Closes https://github.com/flutter/flutter/issues/3824

### CODE REVIEW

**Comment by sethladd** [comment]
cc @devoncarew

File: packages/flutter_tools/lib/src/usage.dart [hunk 1]
```diff
@@ -93,23 +93,18 @@ class Usage {
...
-  ╚════════════════════════════════════════════════════════════════════════════════════════════════════╝
+  ╔════════════════════════════════════════════════════════════════════════════╗
+  ║                 Welcome to Flutter! - https://flutter.io                   ║
+  ║                                                                            ║
+  ║ The Flutter tool anonymously reports feature usage statistics and basic    ║
+  ║ crash reports to Google in order to help Google contribute improvements to ║
+  ║ Flutter over time. See Google's privacy policy:                            ║
+  ║ https://www.google.com/intl/en/policies/privacy/                           ║
+  ║                                                                            ║
+  ║ Use "flutter config --no-analytics" to disable analytics reporting         ║
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/usage.dart]: nit nit: I would center this line, or end it with a period.

**Comment by devoncarew** [comment]
thanks! lgtm

File: packages/flutter_tools/lib/src/usage.dart (hunk 1 above)
Comment (sethladd) [inline packages/flutter_tools/lib/src/usage.dart]: good catch! fixed.

**Comment by sethladd** [comment]
Blocked on https://github.com/flutter/flutter/issues/3894

**Pull request merged**